lib/index.js

```diff
@@ -143,7 +143,9 @@ module.exports.registerEventHandler = registerEventHandler;
 module.exports.add = add;
 module.exports.remove = remove;
 module.exports.skip = TrackPlayer.skip;
+module.exports.getQueue = TrackPlayer.getQueue;
```

Comment:
@Guichaguri added this one. It will return the whole queue. It'll return the same object that was passed in. iOS already saves the original object.